app/javascript/controllers/projects/samples/attachments/concatenations_controller.js

```diff
@@ -0,0 +1,48 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["field"];
+
+  static values = {
+    fieldName: String,
+    storageKey: {
+      type: String,
+      default: location.protocol + "//" + location.host + location.pathname,
+    },
+  };
+
+  connect() {
+    const storageValues = JSON.parse(
+      sessionStorage.getItem(this.storageKeyValue)
+    );
+
+    if (storageValues) {
+      for (let [storageValueIndex, storageValue] of storageValues.entries()) {
+        const value = JSON.parse(storageValue);
+
+        if (value instanceof Array) {
+          for (let arrayValue of value) {
+            const element = document.createElement("input");
+            element.type = "hidden";
+            element.id = this.fieldNameValue + "[" + storageValueIndex + "][]";
```

Comment:
Can you clean up all the string concatenation here with string templates

Reply:
Assuming you're going to take care of the selection controller, see 3ee83cb5365b6845c7ab72299cfab40c81b7543f.

Reply:
Thanks, much easier to read.